susemanager-utils/susemanager-sls/susemanager-sls.spec

```diff
@@ -22,11 +22,11 @@
 %endif
 
 %if 0%{?suse_version}
-%global serverdir  /usr/share/susemanager
-%global wwwdocroot %{serverdir}/www/htdocs
+%global serverdir  /srv
+%global wwwpubroot %{serverdir}/www/htdocs
```

Comment:
Same here I would keep those outside of `/srv` to make container volumes simpler. Otherwise we'll have to cut holes in them for RPM-managed files.

Reply:
This variable is only used in this spec file and avoids hardcoding the same path (`/srv/www/htdocs` ) in line 133.